### PR TITLE
Add method description to `get_current_index` in `PopupMenu`

### PR DESCRIPTION
**Pull request by SirQuartz**
This pull request adds a missing method description to `PopupMenu`.

This completes the documentation for `PopupMenu` and enhances usability by doing so.

### CODE REVIEW

File: doc/classes/PopupMenu.xml [hunk 1]
```diff
@@ -260,6 +260,7 @@
 			<return type="int">
 			</return>
 			<description>
+				Returns the current focused index. Returns -1 if no index is focused.
```
Comment (akien-mga) [inline doc/classes/PopupMenu.xml:263]:
```suggestion
				Returns the current focused index. Returns [code]-1[/code] if no index is focused.
```

**Comment by akien-mga** [comment]
Please squash the commits together.

**Comment by SirQuartz** [comment]
> Please squash the commits together.

Everything should be good now.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.4.

**Comment by akien-mga** [comment]
Cherry-picked for 3.3.3.

**Pull request merged**